ios/FluentUI/PreviewCard/PreviewCard.swift

```diff
@@ -30,36 +30,58 @@ import SwiftUI
 
 /// View that represents the PreviewCard.
 public struct PreviewCard: View {
-
+    var isElevated: Bool
     @Environment(\.theme) var theme: FluentUIStyle
     @Environment(\.windowProvider) var windowProvider: FluentUIWindowProvider?
     /// Creates inner PreviewCard view.
     @ViewBuilder
     var innerContents: some View {
         HStack {
             Spacer()
+                /// Sets the height and width of the Preview Card.
                 .frame(minWidth: Constants.cardMinWidth, minHeight: Constants.cardMinHeight)
-        }
+                /// Sets the background border style, radius, color, and linewidth of the Preview Card.
+                .background(
+                    RoundedRectangle(cornerRadius: Constants.cardCornerRadius)
+                        .strokeBorder(Color(Color.RGBColorSpace.sRGB,
```

Comment:
You should be able to skip this first argument for colorspace, since it's a default value in the API.